src/reaper_osara.cpp

```diff
@@ -3583,6 +3583,26 @@ void cmdUnselAllTracksItemsPoints(Command* command) {
 		outputMessage(translate("unselected tracks/items/envelope points"));
 }
 
+void reportTempoTimeSig() {
+ReaProject* proj=EnumProjects(-1, nullptr, 0);
```

Comment:
While it's not necessary to get the code to compile, code blocks should be indented for good style. We use a tab character for each level of indentation. Unfortunately, we don't have automatic code formatting working yet, so this has to be done manually.